packages/world-modules/src/modules/erc20-puppet/tables/ERC20Metadata.sol

```diff
@@ -29,28 +26,10 @@ library ERC20Metadata {
   FieldLayout constant _fieldLayout =
     FieldLayout.wrap(0x0001010201000000000000000000000000000000000000000000000000000000);
 
-  /**
-   * @notice Get the table's key schema.
-   * @return _keySchema The key schema for the table.
-   */
-  function getKeySchema() internal pure returns (Schema) {
-    SchemaType[] memory _keySchema = new SchemaType[](0);
-
-    return SchemaLib.encode(_keySchema);
-  }
-
-  /**
-   * @notice Get the table's value schema.
-   * @return _valueSchema The value schema for the table.
-   */
-  function getValueSchema() internal pure returns (Schema) {
-    SchemaType[] memory _valueSchema = new SchemaType[](3);
-    _valueSchema[0] = SchemaType.UINT8;
-    _valueSchema[1] = SchemaType.STRING;
-    _valueSchema[2] = SchemaType.STRING;
-
-    return SchemaLib.encode(_valueSchema);
-  }
+  // Hex-encoded key schema of ()
```

Comment:
kinda cursed but what can ya do

Reply:
it's accurate at least!

Reply:
I could add a conditional for this but seems messy